L1Trigger/TrackFindingTracklet/interface/Settings.h

```diff
@@ -638,10 +638,15 @@ namespace trklet {
                                                                   {{6, 6, 6, 6, 10, 10, 10, 10, 0, 0, 6, 0}},
                                                                   {{0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 4, 6}}}};
 
+    /* std::array<std::array<unsigned int, N_SEED>, 3> lutwidthtabextended_{ */
```

Comment:
Why is the lutwidthtabextended changed? And why can't we delete the commented out code?

Reply:
lutwidthtabextended is changed to take into account the fact that the maximum number of bits needed to represent lutval is not 0 for triplet seeds. The width still needs to be optimized before we settle on a final value. 21 was found to be a good maximum number to not face the error complaining of too many bits. (The actual widths should be smaller)

Reply:
Please add two comments. (1) Explain what lutwidthtab*_ in Settings.h represents; (2) Add "TO DO: tune lutwidthtabextended_ values".

Reply:
Done.